docker/stretch-image.docker

```diff
@@ -1,32 +1,34 @@
-FROM debian:jessie-backports
```

Comment:
`jessie-backports` does not exist anymore, the subsequent update returns 404 on repos